datasets/aime-2024/prepare.py

```diff
@@ -0,0 +1,147 @@
+import json
```

Comment:
Please add these two new test files to the tests/test_datasets.py

Reply:
Added